pkg/volume/volume.go

```diff
@@ -33,6 +33,9 @@ type Volume interface {
 
 	// MetricsProvider embeds methods for exposing metrics (e.g. used,available space).
 	MetricsProvider
+
+	// GetPropagationMode returns the propagation mode the volume is mounted to.
+	GetPropagationMode() api.PropagationMode
```

Comment:
Thanks @Kaffa-MY!  
Yes, there some issues to sort out here. To understanding, there are two propsals now: 
  spec.containers.volumeMounts.propagation
  spec.containers.volumes.hostPath.propagation
And you are favoring of the second one. If we put propagation mode in the Attributes, Can we just update the method GetAttributes() in hostPath.go? Because other volume source dose not care for this value.